### ISSUE TITLE
Plugin not respnding

### ISSUE THREAD

**Issue by Cookiemonsterscookie** [marcsowen/homebridge-homematicip#360]
Hello Mr Sowen,

I am a newbie in coding. So thank you in advance for bearing with my stupidity :)

I installed your plugin on my hoobs and set the config provided below:

{
    "platform": "homebridge-homematicip",
    "name": "HomematicIP",
    "access_point": "yaddadadada",
    "pin": "xxxx"
}

Unfortunatly in my log I only get this:
Homematicip BridgeLoaded plugin 'homebridge-homematicip'
Homematicip BridgeLoading 2 platforms...
Homematicip BridgeBridge is running on port 51836.

I already tried these steps:
Restarting Hoobs
Restarting the physical Homematic-IP hub
Reinstalling the plugin
Changing the port
Adding and removing the pin in the homeatic app
Factory reseting Hoobs

A few months ago I have also tried to install your plugin. Maybe I got the auth_token back then. But I never saved it. 

Sincerely
Cookiemonsters cookie

**Comment by marcsowen** [comment]
Hi @Cookiemonsterscookie 

as long as you don't specify "auth_token" in your config, the plugin will try to acquire a new auth_token. So it doesn't really matter if you already have a valid auth_token. You can see all auth tokens in the HomematicIP app (Settings / User overview). You can also remove an auth token by swiping over one of the tokens. The plugin will use "homebridge-homematicip" as the user name.

I wonder why you can't get anymore log output. Can you maybe enable "debug" mode fore homebridge? https://github.com/homebridge/homebridge/wiki/Basic-Troubleshooting#debug-mode

Are you sure "access_point" has the SGTIN of you access point? Preferably without dashes and all capital letters, e.g.

`"access_point": "3014FEAA222211144A009332"`

**Comment by CBHomekitCrazy** [comment]
Hello,

I have exact same problem as above and getting exact same logs. Feels like I have been trying everything. Seems like it just don't see my HomematicIP Access Point, no matter what I try. Debug is enabled, but doesn't change the logs. still same 3 lines. 
Just for the record I have couple of other bridges running for other equipment, that works fine on my HOOBS. everything is on same network incl my HomematicIP Access Point.

**Comment by Devel1990** [comment]
Hello together,
having the same problem as mentioned from @Cookiemonsterscookie  and @CBHomekitCrazy. Wondering if it has connection to recent PIN / user Management change of HomematicIP. I rebooted everything several times, checked the SGTIN and still same lines.

thanks for feedback

**Comment by marcsowen** [comment]
Hi,

I must admit I haven't changed my HmIP cloud to the new user management scheme. Therefore I don't know if the plugin even supports it. (Maybe someone who tried can deny or confirm?) I would guess that after adding a new "Homebrige-HomematicIP-User" to the HmIP cloud you would need to give this user quite a few user rights via the HmIP app to let the plugin control most of your home.

When I find sufficient time I will try to change the user management on my setup and try to add a new user to see if everything works.

Just to confirm: Are you sure that you removed all "auth_token" keys from your config file?

Marc

**Comment by CBHomekitCrazy** [comment]
Hi,In my case yes. No auth_token keys as I never got one.I have tried both with and without enablibg user management. Dont see a difference. Den 14. nov. 2022 kl. 09.50 skrev Marc Sowen ***@***.***>:﻿
Hi,
I must admit I haven't changed my HmIP cloud to the new user management scheme. Therefore I don't know if the plugin even supports it. (Maybe someone who tried can deny or confirm?) I would guess that after adding a new "Homebrige-HomematicIP-User" to the HmIP cloud you would need to give this user quite a few user rights via the HmIP app to let the plugin control most of your home.
When I find sufficient time I will try to change the user management on my setup and try to add a new user to see if everything works.
Just to confirm: Are you sure that you removed all "auth_token" keys from your config file?
Marc

**Comment by Devel1990** [comment]
Hi,
I tried also setting up a user management incl. Pin no change. To get the auth_token using following scheme, as stated:
![grafik](https://user-images.githubusercontent.com/118108937/201785617-fc872d7e-146d-4126-b66d-aebd72af9b39.png)
Logfile:
14.11.2022, 23:56:23Homematicip BridgeLoaded plugin 'homebridge-homematicip'
14.11.2022, 23:56:23Homematicip BridgeLoading 2 platforms...
14.11.2022, 23:56:23Homematicip BridgeBridge is running on port 51846.

Any idea appreciated.
Thanks
Devel

**Comment by marcsowen** [comment]
It might be some logging issue. The relevant messages are logged using log level "info". Maybe these log messages are suppressed on your device. Can you somehow change the homebridge log level? Maybe even start homebridge in "debug mode"?

**Comment by marcsowen** [comment]
@CBHomekitCrazy Why is your platform "homebridge-homematicip"? The platform is called "HomematicIP".

**Comment by CBHomekitCrazy** [comment]
Yep, but this platform text came after installing latest version 1.1.0, with previous version it was "HomematicIP" - I can't change the name. Well I can, but after saving the config and writing platform "HomematicIP" it changes platform back to "homebridge-homematicip". Only name, access_point and pin can be changed and stays after saving

Your support is really appreciated, thx

**Comment by Devel1990** [comment]
I can confirm - same in my case.
@CBHomekitCrazy are you also applying Hoobs? I was wondering if this is the root cause and might change to homebridge to double check on the weekend...

Additional info: I tried older versions like 0.7.2 and 0.8.0, and also there "HomematicIP" will change into "homebridge-homematicip".

Any idea highly appreciated, thx

**Comment by CBHomekitCrazy** [comment]
Yep, using Hoobs kit, only homebridge setup I have … so far.

**Comment by Devel1990** [comment]
Hoobs was the issue, it is working now via homebridge. Thanks for your support Von meinem iPhone gesendetAm 16.11.2022 um 22:56 schrieb CBHomekitCrazy ***@***.***>:﻿
Yep, using Hoobs kit, only homebridge setup I have … so far.

**Comment by marcsowen** [comment]
@Devel1990 Good to hear.

But strange, Hoobs must have changed the way they are naming the plugins. Maybe I am doing something wrong. Maybe someone can ask the Hoobs support for help? If I'm missing something, please let me know.

**Comment by askovi** [comment]
Got pinged about this issue. Looks like this plugin doesn't have a config schema file and HOOBS was trying to compensate for it. I overwrote the config schema in the HOOBS plugin db so this plugin can now be configured visually (For the basic config only). I also fixed the platform naming issue. 
![image](https://user-images.githubusercontent.com/3981552/203446110-12beb3db-7d56-4462-a170-de2c253ad8ea.png)
![image](https://user-images.githubusercontent.com/3981552/203446134-c1c98f9e-61ec-4b4e-b856-a5f442d3d648.png)

**Comment by CBHomekitCrazy** [comment]
@askovi @marcsowen @Devel1990 
Thx to all and Ask Ovi for his fix. Works on Hoobs as well now !! :-)

**Comment by marcsowen** [comment]
I have also added a config schema now. It should enable even more configuration options.
